contracts/Strategy.sol

```diff
@@ -616,52 +704,35 @@ contract Strategy is BaseStrategy {
         return balanceOfInvestmentToken().sub(balancePrior);
     }
 
-    function _depositInvestmentTokenInYVault() internal {
+    function _depositInvestmentTokenInYVault() private {
         uint256 balanceIT = balanceOfInvestmentToken();
         if (balanceIT > 0) {
-            _checkAllowance(
-                address(yVault),
-                address(investmentToken),
-                balanceIT
-            );
```

Comment:
@newmickymousse wouldn't removing the call to `_checkAllowance` break `migrateToNewMIMYVault`? There is no way to reset/redo the approval and `yVault` address would change.

Reply:
Yes, absolutely right.